tests/integration/TEC/Tickets/QR/QR_Test.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace TEC\Tickets\QR;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use tad\WP\Snapshots\WPHtmlOutputDriver;
+
+/**
+ * Class QR_Test.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\QR
+ */
+class QR_Test extends \Codeception\TestCase\WPTestCase {
+	use MatchesSnapshots;
+
+	protected $driver;
+
+	public function setUp(): void {
+		parent::setUp();
+
+		// Setup a new HTML output driver to make sure our stuff is tolerable.
+		$this->driver = new WPHtmlOutputDriver( home_url(), 'http://views.dev' );
```

Comment:
This builds a test dependency on the snapshot framework. 
I would remove this one and go with the general solution of filtering the `home_url` function to return `https://views.dev` to stabilize the snapshots.

Also: do not extend the `setUp` method, use a method with a `@before` annotation for better cross-compatibility with Codeception and PHPUnit versions and better readability.